src/button/tests/unit/Button.ts

```diff
@@ -80,11 +81,7 @@ registerSuite('Button', {
 				value: 'value'
 			}, [
 				'foo',
-				v('i', {
-					classes: [ css.addon, iconCss.icon, iconCss.downIcon ],
-					role: 'presentation',
-					'aria-hidden': 'true'
-				})
+				w(Icon, { type: 'downIcon', extraClasses: { root: css.addon } })
```

Comment:
same comment again re. wrapping the icon with something for the sake of `css.addon`